backend/database/models/incidents.py

```diff
@@ -2,6 +2,8 @@
 from backend.database import db
 
 from flask_serialize.flask_serialize import FlaskSerialize
```

Comment:
If you have Marshmallow then you don't really need/want another serialization library. And Marshmallow is definitely the right choice here. So with the addition of Marshmallow we can/should remove FlaskSerialize.

Reply:
Removed flask-serialize